### PR TITLE
Sorting values

### PR DESCRIPTION
**Pull request by ptbrowne**
1. Dimension values were not parsed when fetched in batch
2. The sorters from makeDimensionValueSorters return sorters that must receive dimension labels, not dimension values

### CODE REVIEW

File: app/configurator/components/field.tsx [hunk 1]
```diff
@@ -702,36 +733,23 @@ export const ChartFieldField = ({
...
-            </span>
-          ) : (
-            <span>{label}</span>
-          )}
-          {fetching ? (
-            <CircularProgress size={12} className={classes.loadingIndicator} />
-          ) : null}
-        </>
+        <FieldLabel
+          isOptional={optional || false}
```
Comment (bprusinowski) [inline app/configurator/components/field.tsx:749]: Nit: not sure if we need `|| false`?

**Review by bprusinowski — APPROVED** [review]
LGTM 👏

File: app/configurator/components/field.tsx (hunk 1 above)
Comment (ptbrowne) [reply app/configurator/components/field.tsx:749]: Ah yes you are right :)

File: app/locales/de/messages.po [hunk 2]
```diff
@@ -13,24 +13,24 @@ msgstr ""
 "Language-Team: \n"
 "Plural-Forms: \n"
 
-#: app/configurator/components/chart-configurator.tsx:701
+#: app/configurator/components/chart-configurator.tsx:713
 msgid "Add filter"
 msgstr "Filter hinzufügen"
```
[the remark this answers is not in the document]
Comment (ptbrowne) [reply app/locales/de/messages.po:19]: Done in 91f376e0

**Pull request merged**